### ISSUE TITLE
Parameters not compatibles

### ISSUE THREAD

**Issue by grosjo** [nextcloud/integration_mattermost#31]
The Nextcloud menu is as follows 
![image](https://github.com/nextcloud/integration_mattermost/assets/2287111/24d04060-e07e-4776-94db-d8abc48294ec)

But those fields are not provided nowhere (Application ID ??? Applicaiton Secret ?? Instance address ??)
I have create a "token"  in the Mattermost, but then this is not the same fields required

**Comment by grosjo** [comment]
On the side of Mattermost, "creating an oauth 2 app" requires other fields

![image](https://github.com/nextcloud/integration_mattermost/assets/2287111/e7390314-374e-497b-8616-07c31c4d5c71)

**Comment by grosjo** [comment]
In all cases, the integration app should rather
1 - get access to Mattermost
2 - let people connect to their Mattermost (login/pass with a token generated automaticallly) from nextcloud

For the admin, a documentation clarifying what fields to put in the numerous pages (toeksn creation, then OAuth 2.0 in Mattermost with unknown URL, then the mistery fields in Nextcloud -> This is the actual purpose of an intergatoin app, right ?

**Comment by kyteinsky** [comment]
Hi, This is a standard OAuth 2.0 flow. If you wish to connect using your personal token (generated on Mattermost) or login id/password, head over to the "Connected accounts" section in the user settings.

**Comment by grosjo** [comment]
But what are the code to enter ???

Mattermost is based on login/pass

**Comment by kyteinsky** [comment]
Yes, it is. However there are other means to prove you are the owner of the account without providing your login/pass.

Login id and password can also be used to login in this integration. You can do the following:
1. Go into personal settings -> Connected Accounts
2. Enter the details: "Mattermost instance address", "Login", and "Password"
3. Click on "Connect to Mattermost"

**Comment by grosjo** [comment]
The connector does not ask for such fields. Not sure what you are referring to....
